configs/milvus.yaml

```diff
@@ -412,7 +412,6 @@ queryNode:
       nprobe: 16 # nprobe to search small index, based on your accuracy requirement, must smaller than nlist
       memExpansionRate: 1.15 # extra memory needed by building interim index
       buildParallelRate: 0.5 # the ratio of building interim index parallel matched with cpu num
-    multipleChunkedEnable: true # Enable multiple chunked search
```

Comment:
should we keep this config and mark it as useless with a comment? If a user upgraded the binary to the new version and kept this config, will this lead to crash?